### DISCUSSION TITLE
Build and Deploy a Modern Web 3.0 Blockchain App  1:31:43

### DISCUSSION THREAD

**Discussion by usayk** [adrianhajdin/project_web3.0#117] [Q&A]
PS D:\BBcyrpto\smart_contract> npm install ganache
/g7 for details

added 330 packages, removed 3 packages, changed 5 packages, and audited 1075 packages in 

93 packages are looking for funding
  run `npm fund` for details

29 vulnerabilities (21 moderate, 7 high, 1 critical)
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
PS D:\BBcyrpto\smart_contract> npx hardhat run scripts/deploy.js --network goerli        
Error HH100: Network goerli doesn't exist

PS D:\BBcyrpto\smart_contract> npx hardhat init run scripts/deploy.js --network goerli   
Error HH23: You are trying to initialize a project inside an existing Hardhat project. The path to the project's configuration file is:  D:\BBcyrpto\smart_contract\hardhat.config.js.

PS D:\BBcyrpto\smart_contract> npx hardhat run scripts/deploy.js --show-stack-traces goerli
Error HH308: Unrecognized positional argument goerli

HardhatError: HH308: Unrecognized positional argument goerli
    at ArgumentsParser._parsePositionalParamArgs (D:\BBcyrpto\smart_contract\node_modules\hardhat\src\internal\cli\ArgumentsParser.ts:351:13)
    at ArgumentsParser.parseTaskArguments (D:\BBcyrpto\smart_contract\node_modules\hardhat\src\internal\cli\ArgumentsParser.ts:183:38)
    at main (D:\BBcyrpto\smart_contract\node_modules\hardhat\src\internal\cli\cli.ts:306:39)
PS D:\BBcyrpto\smart_contract> npx hardhat init run scripts/deploy.js --network ropsten  
Error HH23: You are trying to initialize a project inside an existing Hardhat project. The path to the project's configuration file is:  D:\BBcyrpto\smart_contract\hardhat.config.js.

For more info go to https://hardhat.org/HH23 or run Hardhat with --show-stack-traces     
PS D:\BBcyrpto\smart_contract> npx hardhat init run scripts/deploy.js --network goerli   
Error HH23: You are trying to initialize a project inside an existing Hardhat project. The path to the project's configuration file is:  D:\BBcyrpto\smart_contract\hardhat.config.js.

For more info go to https://hardhat.org/HH23 or run Hardhat with --show-stack-traces     
PS D:\BBcyrpto\smart_contract> 
 *  History restored

**Comment by usayk** [comment]
when i write =>   npx hardhat run scripts/deploy.js --network goerli i taked this warning . can you halp me sir pls 🧇

**Reply by aScHEABL** [reply]
Did you check your deployed app on [Alchemy](https://www.alchemy.com/)?
I assumed you chose something other than goerli there?
If not check my [post](https://github.com/adrianhajdin/project_web3.0/discussions/116).

On the side note, I've checked your github profile and found out you're a Japanese.
Just a friendly reminder that you've made numerous typos and spelling errors in your projects and intro.
As you probably haven't realized that.
Eg. Android Developer And Coder For "Everyting"

I pointed this out because you probably have done a similar typo like this somewhere in your codebase, that's why you ran into development issue.
